chef_dsl_metadata/README.md

```diff
@@ -1,8 +1,8 @@
 # Generating metadata for a specific chef version
 - run 'bundle install' to install appraisal
 - update Appraisals file with new chef version(s)
-- run 'appraisal chef_12.7.2 bundle install' to generate the Gemfile and Gemfile.lock
-- run 'appraisal chef_12.7.2 rake generate_chef_metadata' to generate json for that version of chef
+- run 'bundle exec appraisal install'. It will fail, but it will still generate the Gemfile
```

Comment:
The bundle install command fails as the gem isn't there yet.  The appraisal install command generates all the Gemfiles.

```
appraisal chef_12.8.1 bundle install
WARN: Unresolved specs during Gem::Specification.reset:
      rake (>= 0)
WARN: Clearing out unresolved specs.
Please report a bug if this causes problems.
>> BUNDLE_GEMFILE=/Users/tsmith/dev/oss/foodcritic_mine/chef_dsl_metadata/gemfiles/chef_12.8.1.gemfile bundle install
/Users/tsmith/dev/oss/foodcritic_mine/chef_dsl_metadata/gemfiles/chef_12.8.1.gemfile not found
```